worlds/pokemon_rb/options.py

```diff
@@ -10,18 +10,20 @@ class GameVersion(Choice):
     default = "random"
 
 
-class TrainerName(FreeText):
-    """Your trainer name. Cannot exceed 7 characters.
+class TrainerName(TextChoice):
+    """Your trainer name. If not set to choose_in_game, must be a string not exceeding 7 characters.
```

Comment:
wonder if we should say "text" rather than "string", given that it's user facing.

Reply:
reworded